### ISSUE TITLE
Error after rebooting kubernetes minion

### ISSUE THREAD

**Issue by olalonde** [deis/deis#5076]
I rebooted one of my k8s minions a few hours ago and am now getting now seeing "RunContainerError" on one of my apps. I'm also seeing this with describe pods:

```
Events:
  FirstSeen     LastSeen        Count   From                                                    SubobjectPath               Type             Reason          Message
  ---------     --------        -----   ----                                                    -------------               -------- ------          -------
  3m            3m              1       {default-scheduler }                                                                Normal           Scheduled       Successfully assigned blockai-api-v13-web-s71rt to ip-172-20-0-131.us-west-1.compute.internal
  3m            12s             10      {kubelet ip-172-20-0-131.us-west-1.compute.internal}    spec.containers{blockai-api-web}     Normal          Pulling         pulling image "quay.io/deis/slugrunner:v2.2.0"
  3m            11s             10      {kubelet ip-172-20-0-131.us-west-1.compute.internal}    spec.containers{blockai-api-web}     Normal          Pulled          Successfully pulled image "quay.io/deis/slugrunner:v2.2.0"
  2m            7s              10      {kubelet ip-172-20-0-131.us-west-1.compute.internal}                                Warning          FailedSync      Error syncing pod, skipping: failed to "StartContainer" for "blockai-api-web" with RunContainerError: "GenerateRunContainerOptions: Couldn't find key port in Secret blockai-api/blockai-api-v13-env"
```

**Comment by bacongobbler** [comment]
IIRC this was fixed in v2.3.0. Can you confirm? Sorry about the delay on this one. In the future if you can post issues for v2 on deis/controller we'd be able to pick up on it quicker. Thanks! :)

**Comment by olalonde** [comment]
Ok thanks, I will upgrade to v2.3.0 later. Haven't seen that error since I set PORT in my config (https://github.com/deis/workflow/issues/407), not sure if that issue was related. Will re-open if I get this issue again.
